### DISCUSSION TITLE
Jonathan, Sturmer, Green Cheese and Gravenstein are varieties of which fruit?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#199] [Galaxy Trivia] [answered]
**Hint:** keeps doctor away

**Comment by MarieCamillePetit** [comment, marked as answer]
Apple ! 🍎

**Reply by ParanoidUser** [reply]
Lovely! Congratulations on your well deserved [Galaxy Brain](https://github.com/MarieCamillePetit?tab=achievements&achievement=galaxy-brain) badge. Continue to unlock the next badge levels (Bronze for 4 answers), and don't hesitate to post your own questions to the community. 😊🎉
